docker-compose.yml

```
@@ -81,14 +81,17 @@ services:
       - "${MISEREND_PORT:-8000}:8000"
     environment:
         MISEREND_WEBAPP_ENVIRONMENT: ${MISEREND_WEBAPP_ENVIRONMENT:-development}
+        MISEREND_MYSQL_DATABASE:  ${MISEREND_MYSQL_DATABASE}
+        MISEREND_MYSQL_USER:  ${MISEREND_MYSQL_USER}
+        MISEREND_MYSQL_PASSWORD:  ${MISEREND_MYSQL_PASSWORD}
     networks:
       inner:
         aliases:
           - miserend
     depends_on:
       - mysql
       - mailcatcher
-      - build
+      - miserend-build
```

```suggestion
      - build
```

igyekezzünk ne keverni a dolgokat

---

#233 megjavítja